### PR TITLE
noise profiles

### PR DESCRIPTION
**Pull request by rawfiner**

### CODE REVIEW

File: data/noiseprofiles.json [hunk 1]
```diff
@@ -8527,6 +8550,50 @@
...
+            {"name": "D-Lux 7 iso 10000", "iso": 10000, "a": [0.00036243475124656, 0.000118653768298497, 0.000245138702803466], "b": [2.18094201528324e-06, 1.51197754569152e-06, 2.53047479139305e-06]},
+            {"name": "D-Lux 7 iso 12500", "iso": 12500, "a": [0.000449640776725114, 0.000148446440408457, 0.000311886091782767], "b": [2.37996955034241e-06, 2.00015896812575e-06, 2.88449986496872e-06]},
+            {"name": "D-Lux 7 iso 16000", "iso": 16000, "a": [0.000283067530749418, 9.35090443171114e-05, 0.00019806078346406], "b": [1.05913275268099e-06, 8.56211171321623e-07, 1.24206348899041e-06]},
+            {"name": "D-Lux 7 iso 20000", "iso": 20000, "a": [0.000357916782226216, 0.000118289345484384, 0.000249054892994411], "b": [1.59063650580094e-06, 1.29866798859529e-06, 1.8919995222713e-06]},
+            {"name": "D-Lux 7 iso 25000", "iso": 25000, "a": [0.000447900876269519, 0.000148891527550444, 0.000312481699207148], "b": [2.39306717704209e-06, 1.97777706381915e-06, 2.90082699795567e-06]}
+          ]
+        },
+        {
+          "comment": "leica m10 contributed by mandre",
+          "model": "LEICA M10",
```
Comment (kmilos) [inline data/noiseprofiles.json:8583]: Don't think this one'll work: there is no entry for the M10 in rawspeed/data/cameras.xml so maker/model are not "normalized" properly: original maker is "Leica Camera AG" in the Exif, and not translated to "Leica" expected here.

Comment (rawfiner) [reply data/noiseprofiles.json:8583]: I tested it on a raw file, and the noise profile is correctly detected

Comment (kmilos) [reply data/noiseprofiles.json:8583]: Must be doing partial string matching on the maker then...

In any case, would be good to go the normalized rawspeed route and avoid the mix.

**Review by TurboGit — APPROVED** [review]
Thanks !

**Comment by kmilos** [comment]
NB: we'll have to change to "M10" after next rawspeed update, so maybe keep https://github.com/darktable-org/darktable/issues/10122 open as reminder?

**Comment by TurboGit** [comment]
> NB: we'll have to change to "M10" after next rawspeed update, so maybe keep #10122 open as reminder?

Done.

BTW, @kmilos can you share some information about rawspeed updates? What branch should we take for dt? I have hard time to get a proper view about all this from Roman. TIA.

**Comment by kmilos** [comment]
Unfortunately I have the same view as you - just what gets merged in to the develop branch... I have no clue if there is extra testing before porting to stable, and also no idea about that schedule. I did see a recent comment from Roman in one of the issues that he was unavailable to work on rawspeed in general lately :(

**Pull request merged**